corehq/ex-submodules/casexml/apps/case/tests/test_factory.py

```diff
@@ -105,7 +105,7 @@ class CaseFactoryTest(TestCase):
     def test_simple_create(self):
         factory = CaseFactory()
         case = factory.create_case()
-        self.assertIsNotNone(CaseAccessors().get_case(case.case_id))
+        self.assertIsNotNone(CaseAccessors(case.domain).get_case(case.case_id))
```

Comment:
nit: since this is a test, seems ok, but passing case.domain instead of the domain name directly, just beats the check for domain.